osmtm/tests/test_user.py

```diff
@@ -16,7 +16,7 @@ def test_users__logged_in(self):
 
     def test_users_json(self):
         res = self.testapp.get('/users.json', status=200)
-        self.assertEqual(len(res.json), 5)
+        self.assertEqual(len(res.json), 7)
```

Comment:
I think tests run alphabetically? This didn't fail before because the test function I originally added was `test_verified_statuses`. Now that it's `test_editor...`, it runs before `test_users_json`, adding two new users and throwing off this count.